tests/ee/roles/current_container_facts/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Retrieve information on current container
+  community.docker.current_container_facts:
+  register: result
+
+- name: Print facts returned by module
+  debug:
+    var: result.ansible_facts
+
+# FIXME: For some reason, the module doesn't detect that it is running inside a container?!
+#- name: Validate results
+#  assert:
+#    that:
+#      - ansible_module_running_in_container
+#      - ansible_module_container_type != ''
```

Comment:
I 'simply' fixed it instead ;-) It was a bit more tricky, GHA has a special way of running Docker containers (I had to enhance current_container_facts for that), and also I realized that it couldn't succeed in the first place since it wasn't running Docker but podman... :)

Reply:
interesting!